packages/astropy/meta.yaml

```diff
@@ -0,0 +1,12 @@
+package:
+  name: astropy
+  version: 3.1.2
+
+source:
+  url: https://pypi.io/packages/source/a/astropy/astropy-3.1.2.tar.gz
+  sha256: 4a78a8ec9666d0a51a37f03494aaa5012e241ba37053e6c913c039cddee89ede
+
```

Comment:
You'll want to put:

```
requirements:
  run:
    - numpy
```

here so that it knows it requires numpy.

Reply:
Hey @joseph-long - are you interested in trying out this fix?

Reply:
Yes, probably still worth a try. I'd need to get my pyodide build env up and running again though... I don't remember now if there was another issue that presented itself after trying @mdboom's suggestion that led me to abandon the attempt.